lib/templates/layout-owee/partials/_footer.tpl

```diff
@@ -1,9 +1,9 @@
 <!-- Scripts -->
-<script src="/layout-owee/js/skel.min.js"></script>
-<script src="/layout-owee/js/jquery.scrollex.min.js"></script>
-<script src="/layout-owee/js/util.js"></script>
-<!--[if lte IE 8]><script src="/layout-owee/js/ie/respond.min.js"></script><![endif]-->
-<script src="/layout-owee/js/main.js"></script>
+<script src="/assets/layout-owee/js/skel.min.js"></script>
+<script src="/assets/layout-owee/js/jquery.scrollex.min.js"></script>
+<script src="/assets/layout-owee/js/util.js"></script>
+<!--[if lte IE 8]><script src="/assets/layout-owee/js/respond.js"></script><![endif]-->
```

Comment:
Meh, te veel moeite om ie8 en onder te ondersteunen. Waarschijnlijk gaat het in IE8 wel op een andere plek stuk.